usr/share/rear/lib/global-functions.sh

```diff
@@ -228,6 +247,23 @@ mount_url() {
 	(sshfs)
 	    mount_cmd="sshfs $(url_host $url):$(url_path $url) $mountpoint -o $options"
             ;;
+        (ftpfs)
+            local hostname=$( url_hostname $url )
+            test "$hostname" || Error "Cannot run 'curlftpfs' because no hostname found in URL '$url'."
+            local path=$( url_path $url )
+            test "$path" || Error "Cannot run 'curlftpfs' because no path found in URL '$url'."
+            local username=$( url_username $url )
+            if test "$username" ; then
+                local password=$( url_password $url )
+                if test "$password" ; then
+                    mount_cmd="curlftpfs -o user=$username:$password ftp://$hostname$path $mountpoint"
```

Comment:
@jsmeix it could become more complex I'm afraid if we are working with SSL certificates. We should mentioned this when detected, or what do you propose?